content/About Arduino/Trademarks, Logos and Licenses/Arduino-and-Community-logos.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Arduino and Community logos"
+---
+
+This article covers the basic trademark rules that must be followed when using the [Arduino Logo](#arduino-logo) and the [Community Logo](#community-logo).
+
+---
+
+<h2 id="arduino-logo">Arduino Logo</h2>
+
+The Arduino logo can be found in the following colors:
+
+![Black Arduino Infinity Logo](img/ArduinoLogo_Black.jpg) ![Blue Arduino Infinity Logo](img/ArduinoLogo_Blue.jpg)
+
+ > The logo can sometimes be accompanied by the word mark *Arduino* or appear on its own.
+
+The Arduino logo is a registered trademark and cannot be used by third parties without the trademark owner’s authorization. Generally speaking, you **cannot** use the Arduino Logo (with or without the word *Arduino*) to identify or advertise your products and/or services. This includes:
+
+* Compatible products <!-- link to Compatible products section-->
+* Publications <!-- link to Courses, workshops and publications section-->
+* Online content.
+
+> Exception when the Arduino Logo is used in a descriptive or explanatory way. For example, the Arduino Logo may be used within a text explaining what Arduino is.
+
+Please be aware that even modified/customized versions of the Arduino Logo are considered trademark violations.
+
+* [See examples](#examples) of correct and incorrect uses of Arduino logos. Note that the examples are for illustrative purposes.
+
+<h2 id="community-logo">Community Logo</h2>
+
+The Arduino Community Logo ([download here](https://www.arduino.cc/en/Trademark/CommunityLogo)) has been developed by Arduino to give users the chance to identify non commercial pages, groups, events and initiatives connected to Arduino.
```

Comment:
```suggestion
The Arduino Community Logo ([download here](https://www.arduino.cc/en/Trademark/CommunityLogo)) has been developed by Arduino for users to identify non-commercial pages, groups, events, and initiatives related to Arduino.
```
For me, "connected" seems to imply some official endorsement, which I don't think is the case.